tests/base.py

```diff
@@ -17,28 +19,58 @@ class PyangBindTestCase(unittest.TestCase):
   pyang_flags = []
   split_class_dir = False
   module_name = 'bindings'
+  remote_yang_files = None
+
+  @classmethod
```

Comment:
(no action required) Thanks for implementing this -- in the future one proposal we had here was either vendoring these files, or importing a known version, but I can implement this later. The current approach means that we get test failures when the source YANG files change, rather than when the library is broken.